test/run_pass/test_tls.js

```diff
@@ -30,7 +30,7 @@
 
  var options = {
    key: fs.readFileSync('resources/my_key.pem').toString(),
-   cert: fs.readFileSync('resources/my_crt.pem').toString(),
+   cert: fs.readFileSync('resources/my_crt.pem'),
```

Comment:
The other `toString` should also be removed, or not?

Reply:
This way both functions calls are tested. So it is like that.

Reply:
Ok, cool.